modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNodeHeader.scss

```diff
@@ -27,3 +42,12 @@ hr {
 		width: 10rem;
 	}
 }
+
+.lfr-objects__model-builder-node-background-icon-container {
```

Comment:
Hey @cgmonte, I think we can rename this class to `lfr-objects__model-builder-node-header-background-icon-container-link` and then use the `.lfr-objects__model-builder-node-header `above to put this type inside it, like:

```scss
.lfr-objects__model-builder-node-header {
	&-background-icon-container-link {
		display: inline-block;
		margin-right: 0.1rem;
		padding-left: 0.15rem;
		padding-top: 0.3rem;
	}
```